### PR TITLE
Searchable dropdown

### PR DESCRIPTION
**Pull request by brian-seruwuge**
Review

Substitute Dropdown with SearchableDropdown

### CODE REVIEW

**Comment by brian-seruwuge** [comment]
EDFui

**Comment by brian-seruwuge** [comment]
ECc

**Comment by brian-seruwuge** [comment]
EDFui

**Comment by brian-seruwuge** [comment]
EDFc

File: src/views/components/ClaimEvaluation/index.tsx [hunk 1]
```diff
@@ -133,23 +127,19 @@ class ClaimEvaluation extends React.Component<Props, State> {
       >
         <Form.Field>
           <label>Agents</label>
-          <Dropdown
+          <SearchableDropdown
             placeholder="Select Agent"
             fluid={true}
```
Comment (epicallan) [inline src/views/components/ClaimEvaluation/index.tsx]: `fluid` is already set to true within `SearchableDropdown` no need to reassign it.

File: src/views/components/ClaimEvaluation/index.tsx [hunk 2]
```diff
@@ -133,23 +127,19 @@ class ClaimEvaluation extends React.Component<Props, State> {
...
-            search={true}
-            selection={true}
             options={this.agentOptions(this.props.agents)}
```
Comment (epicallan) [inline src/views/components/ClaimEvaluation/index.tsx]: There is a `mkOptionsFromUser` function within `lib/helpers.ts` use that one and delete `this.agentOptions`

File: src/views/components/ClaimEvaluation/index.tsx [hunk 3]
```diff
@@ -133,23 +127,19 @@ class ClaimEvaluation extends React.Component<Props, State> {
...
         </Form.Field>
         <Form.Field>
           <label>Type of claim</label>
-          <Dropdown
+          <SearchableDropdown
             placeholder="Select claim"
             fluid={true}
-            search={true}
-            selection={true}
             options={this.claimTypeOptions(this.props.claimTypes)}
```
Comment (epicallan) [inline src/views/components/ClaimEvaluation/index.tsx]:
```suggestion
            values={this.props.claimTypes}
```

Comment (epicallan) [reply src/views/components/ClaimEvaluation/index.tsx]: Delete `this.claimTypeOptions`  function

File: src/views/components/EvaluationDataForm/index.tsx [hunk 4]
```diff
@@ -161,12 +161,10 @@ export default class EvaluationDataForm extends React.Component<Props, State> {
         <Form.Group widths="equal">
           <Form.Field inline={true}>
             <label> Agent Names</label>
-            <Dropdown
+            <SearchableDropdown
               name="agentName"
               placeholder="Select an Agent"
               fluid={true}
```
Comment (epicallan) [inline src/views/components/EvaluationDataForm/index.tsx]: remove

File: src/views/components/EvaluationDataForm/index.tsx [hunk 5]
```diff
@@ -161,12 +161,10 @@ export default class EvaluationDataForm extends React.Component<Props, State> {
...
-              search={true}
-              selection={true}
               options={this.agentNameOptions(this.props.agents)}
```
Comment (epicallan) [inline src/views/components/EvaluationDataForm/index.tsx]:
```suggestion
              options={this.mkOptionsFromUser(this.props.agents)}
```

File: src/views/components/ClaimEvaluation/index.tsx [hunk 6]
```diff
@@ -117,6 +81,14 @@ class ClaimEvaluation extends React.Component<Props, State> {
     this.setState({ claim });
   };
 
+  public mkOptionsFromUser(agents): ClaimsEvaluation[] {
```
Comment (epicallan) [inline src/views/components/ClaimEvaluation/index.tsx]: Noo, this function is already defined in the helper.ts. Why redefine it ??? Just import it in.

File: src/views/components/EvaluationDataForm/index.tsx [hunk 7]
```diff
@@ -150,6 +132,14 @@ export default class EvaluationDataForm extends React.Component<Props, State> {
     }
   };
 
+  public mkOptionsFromUser(agents): Evaluation[] {
```
Comment (epicallan) [inline src/views/components/EvaluationDataForm/index.tsx]: Noo, this function is already defined in the helper.ts. Why redefine it ??? Just import it in.

**Comment by epicallan** [comment]
@brian-seruwuge 

> Within components/DataTable there are Form.Select elements that can be replaced with SearchableDropdown component

Why haven't worked in converting the  `Form.Select ` into `SearchableDropdown` yet ??

**Pull request closed without merging**